block.go

```diff
@@ -39,24 +39,16 @@ func (bc *BlockClient) GetChildren(ctx context.Context, id BlockID, pagination *
 		Object     ObjectType `json:"object"`
 		NextCursor string     `json:"next_cursor"`
 		HasMore    bool       `json:"has_more"`
-		Results    []map[string]interface{}
+		Results    Blocks     `json:"results"`
 	}
 	err = json.NewDecoder(res.Body).Decode(&response)
 	if err != nil {
 		return nil, err
 	}
-	results := make([]Block, len(response.Results))
-	for i, r := range response.Results {
-		b, err := decodeBlock(r)
-		if err != nil {
-			return nil, err
-		}
-		results[i] = b
-	}
 
 	return &GetChildrenResponse{
 		Object:     response.Object,
-		Results:    results,
+		Results:    []Block(response.Results),
```

Comment:
It seems like now we can unmarshal the response into the `GetChildrenResponse` struct directly. We don't need the `response` struct anymore

Reply:
Hopefully the change resolves this.